src/modules/acbs/known-errors.ts

```diff
@@ -21,7 +21,7 @@ export const getDealNotFoundKnownAcbsError = (dealIdentifier: string): KnownErro
 });
 
 export const getFacilityNotFoundKnownAcbsError = (facilityIdentifier: string): KnownError => ({
-  substringToFind: 'The facility not found',
+  substringToFind: 'acility not found',
```

Comment:
Facility please.

Reply:
We could change this to "facility" and make search case insensitive.

Reply:
For some endpoints ACBS returns 'The facility not found' and for others it returns 'Facility not found', so we need to match both. However, I agree with @avaitonis that we could make search case insensitive as I don't see any reason why we need case sensitivity. I've changed the code to make the matching case insensitive.